submission/submission-handlers/src/main/resources/application.yml

```diff
@@ -1,8 +1,8 @@
 spring:
   datasource:
-    url: # Database connection URL
-    username:
-    password:
+    url: jdbc:mysql://mysql-fg-biostudy-dev.ebi.ac.uk:4639/biostd_dev
```

Comment:
Please remove the connection strings and all other data from the configuration file